src/test/java/com/hyperwallet/clientsdk/util/HyperwalletApiClientTest.java

```diff
@@ -2123,7 +2123,7 @@ public void testPutMultipartProxy_withIncorrectPort() {
             hyperwalletApiClient.put(baseUrl + "/test?test-query=test-value", requestBody, TestBody.class);
             fail("Expect HyperwalletException");
         } catch (HyperwalletException e) {
-            assertThat(e.getMessage(), is(containsString("java.net.ConnectException: Connection refused (Connection refused)")));
+            assertThat(e.getMessage(), is(containsString("java.net.ConnectException: Connection refused")));
```

Comment:
Is this change specifically for Java 13? Do we need to make the same change for V3 too?

Reply:
This has already been addressed, but you are right